pjmedia/src/pjmedia-audiodev/wmme_dev.c

```diff
@@ -330,20 +330,19 @@ static void build_dev_info(UINT deviceId, struct wmme_dev_info *wdi,
 
     /* Device Name */
     if (deviceId==WAVE_MAPPER) {
-        strncpy(wdi->info.name, "Wave mapper", sizeof(wdi->info.name));
-        wdi->info.name[sizeof(wdi->info.name)-1] = '\0';
+        pj_ansi_strxcpy(wdi->info.name, "Wave mapper", 
+                             sizeof(wdi->info.name));
     } else {
         const pj_char_t *szPname = WIC_WOC(wic, woc, szPname);
         PJ_DECL_ANSI_TEMP_BUF(wTmp, sizeof(wdi->info.name));
         
-        strncpy(wdi->info.name, 
+        pj_ansi_strxcpy(wdi->info.name, 
                 PJ_NATIVE_TO_STRING(szPname, wTmp, PJ_ARRAY_SIZE(wTmp)),
                 sizeof(wdi->info.name));
```

Comment:
a code indent problem

Reply:
intentional to minimize code change (re: git blame)